Sources/Plasma/FeatureLib/pfConsole/pfConsoleCommands.cpp

```diff
@@ -4078,11 +4078,8 @@ namespace plWaveCmd {
 
 typedef void PrintFunk(const char* str);
 
-static inline float FracToPercent(float f) { return (float)(1.e2 * f); }
-static inline float PercentToFrac(float f) { return (float)(1.e-2 * f); }
-
-static inline float RadToDeg(float r) { return r * 180.f / M_PI; }
-static inline float DegToRad(float d) { return d * M_PI / 180.f; }
+static constexpr float FracToPercent(float f) { return 1.e2f * f; }
+static constexpr float PercentToFrac(float f) { return 1.e-2f * f; }
```

Comment:
Is there a good reason not to significantly improve readability here by changing this to `100.f * f` and `f / 100.f`?